scripts/release-post.js

```diff
@@ -32,16 +32,21 @@ function sendRequest (opts) {
     const options = Object.assign({
       headers: { 'User-Agent': 'nodejs.org release blog post script' }
     }, opts)
-
-    request(options, (err, res, body) => {
+    fetch(options.url, options).then(resp => {
```

Comment:
`fetch()` returns a `Promise` so there is no need to wrap it in `new Promise()`, you can make `sendRequest` return the fetch promise directly.